integration/quarkus/deployment/src/main/java/com/blazebit/persistence/integration/quarkus/deployment/BlazePersistenceProcessor.java

```diff
@@ -76,17 +73,6 @@
     private static final DotName BLAZE_PERSISTENCE_INSTANCE_REPEATABLE_CONTAINER = DotName
             .createSimple(BlazePersistenceInstance.List.class.getName());
 
-    @BuildStep
-    CapabilityBuildItem capability() {
-        String version = Version.getVersion();
-        int dotIndex = version.indexOf('.');
-        // As of version Quarkus 2, the capabilities are read from the extension descriptor
-        if (dotIndex == -1 || Integer.parseInt(version.substring(0, dotIndex)) >= 2) {
-            return null;
-        }
-        return new CapabilityBuildItem(CAPABILITY);
-    }
```

Comment:
I don't think it's worth keeping that for new versions given Quarkus 2 is there for quite a while now and there's very little chance people would use new versions of the extension with it.